worker/src/RTC/RtpPacket.cpp

```diff
@@ -9,6 +9,15 @@
 
 namespace RTC
 {
+	/* Static Class methods. */
+
+	void RtpPacket::Deallocate(RtpPacket* packet)
+	{
+		// Destroy and deallocate the RtpPacket.
+		packet->~RtpPacket();
```

Comment:
@jmillan  I think [destroy](https://en.cppreference.com/w/cpp/memory/allocator_traits/destroy) is preferable then direct destructor call.
```
RtpPacket::AllocatorTraits::destroy(RtpPacket::Allocator::Pool, packet);
```